scripts/exglobal_stage_ic.sh

```diff
@@ -2,19 +2,11 @@
 
 source "${HOMEgfs}/ush/preamble.sh"
 
-# Source FV3GFS workflow modules
-. ${HOMEgfs}/ush/load_fv3gfs_modules.sh
-status=$?
-[[ ${status} -ne 0 ]] && exit ${status}
 
-export job="coupled_ic"
-export jobid="${job}.$$"
-
-# Execute the JJOB
-
-source "${HOMEgfs}/ush/jjob_header.sh" -e "coupled_ic" -c "base coupled_ic"
+source "${HOMEgfs}/ush/jjob_header.sh" -e "stage_ic" -c "base stage_ic"
```

Comment:
This was already done in the j-job.  So, it does not belong here.

Reply:
I'll take out from exglobal_stage file then. Yes also there in j-job. let me check and run again